browser/components/BoardShowPage/MenuSideBar/LabelsPane.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react'
+import SearchBar from './SearchBar'
+import Link from '../../Link'
+import ActionsMenuPane from '../../ActionsMenuPane'
+import commands from '../../../commands'
+// TODO: import './LabelMenu.sass' or created the needed sass for this component
+
+export default class LabelsPane extends Component {
+
+  static propTypes = {
+    board: React.PropTypes.object.isRequired,
+    onClose: React.PropTypes.func.isRequired
+  }
+
+  constructor(props){
+    super(props)
+    this.state = {
+      //states for this component
+      searchTerm: ''
+    }
+    // bind properties like this:
+    this.setSearchTerm = this.setSearchTerm.bind(this)
+  }
+
+  setSearchTerm(event) {
+    this.setState({
+      searchTerm: event.target.value
+    })
+  }
+
+  editLabel(labelId, event){
+    event.preventDefault()
+    this.props.editLabel(labelId)
+    this.props.goToPane('Create Label Pane')()
+  }
+
+  addOrRemoveLabel(labelId, event) {
+    event.preventDefault()
+    commands.addOrRemoveLabel(this.props.card.id, labelId)
+  }
+
+  render() {
+    const { card, board } = this.props
+    const boardLabels = board.labels.map( label => {
+      const checked = card.label_ids.includes(label.id)
+      return <LabelRow
+        key={label.id}
+        checked={checked}
+        label={label}
+        onEdit={this.editLabel.bind(this, label.id)}
+        onClick={this.addOrRemoveLabel.bind(this, label.id)}
+      />
+    })
+    const LabelRow = (props) => {
+      const {label, onClick, onEdit, checked} = props
+
+      return <div className="LabelMenu-LabelRow">
+        <div className="LabelMenu-LabelRow-box" onClick={onClick}>
+          <CardLabel key={label.id} color={label.color} text={label.text} checked={checked}/>
```

Comment:
This may be a missing dependency? I don't see it imported/declared in this module.